source/config/Config.cpp

```diff
@@ -113,7 +113,19 @@ bool PlainConfig::LoadFromJson(const Crt::JsonView &json)
     {
         if (!json.GetString(jsonKey).empty())
         {
-            rootCa = FileUtils::ExtractExpandedPath(json.GetString(jsonKey).c_str());
+            basic_string<char> path = FileUtils::ExtractExpandedPath(json.GetString(jsonKey).c_str());
```

Comment:
Can you add a test for this as well?

Reply:
I'm actually appending it to the return value of GetString to pass to ExtractExpandedPath.